api/api.go

```diff
@@ -15,6 +15,7 @@ import (
 type TrentoApiService interface {
 	IsWebServerUp() bool
 	GetChecksSettingsById(id string) (*webApi.JSONChecksSettings, error)
+	GetClustersSettings() (webApi.ClustersSettingsResponse, error)
```

Comment:
The `GetChecksSettingsById` is not used any longer, and it will be replaced in fact in some other PR?
Can we synchronize this to not have zombie methods here?

Reply:
@arbulu89 Good call, I will remove it in my PR